src/QsCompiler/Tests.Compiler/AccessModifierTests.fs

```diff
@@ -1,53 +1,58 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+﻿// Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
-namespace Microsoft.Quantum.QsCompiler.Testing
+module Microsoft.Quantum.QsCompiler.Testing.AccessModifierTests
```

Comment:
Is there a reason for some of the test files being kept as namespaces and some being changed to modules?

Reply:
Basically, there is no reason for any of these tests to be in a class, because the classes have no fields. A module is simpler and more idiomatic F#. But going from class to module is a big diff, so I only did it for small files where I was already changing most of it anyway.

Reply:
Thank you for the explanation!